src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html

```diff
@@ -88,6 +88,65 @@
           </div>
         </div>
 
+        <!-- Mirroring -->
+        <div class="form-group row">
+          <div class="cd-col-form-offset">
+            <div class="custom-control custom-checkbox">
+              <input type="checkbox"
+                     class="custom-control-input"
+                     id="mirroring"
+                     name="mirroring"
+                     (change)="setMirrorMode()"
+                     [(ngModel)]="mirroring && this.currentPoolName"
+                     formControlName="mirroring">
+              <label class="custom-control-label"
+                     for="mirroring">Mirroring</label>
+              <cd-help-text *ngIf="currentPoolMirrorMode === 'disabled'">
+                <span i18n>You need to enable a <b>mirror mode</b> in the selected pool. Please <a [routerLink]="['/block/mirroring', {outlets: {modal: ['edit', currentPoolName]}}]">click here to select a mode and enable it in this pool.</a></span>
```

Comment:
Put this in a `cd-alert-panel` as a info message and give a click button. 
```suggestion
                <span i18n>You need to enable a <b>mirror mode</b> in the selected pool. Please click on Enable Mirroring mode to select a mode and enable it in this pool, <Enable Mirror mode></span>
```

Lets try to open the modal in the page itself rather than redirecting as user looses the values it has entered on click.